### DISCUSSION TITLE
AGS - Aylur's GTK Shell development

### DISCUSSION THREAD

**Discussion by JaKooLit** [JaKooLit/Hyprland-Dots#312] [Q&A]
@SherLock707 

as you have requested, I created another branch for AGS development...

Right now, I took the development branch ... Im still contemplating if move to full AGS or will leave a dual option, or maybe in tandem, where could be part of a choice or something :)

When you got a chance, can you drop me a message on discord?.. I just want to ask something from you.

**Comment by SherLock707** [comment]
Awesome! Lets do this! ✨

_(When I said ags tag actually I meant in your repo description tags, but I guess we need this too!)_
